geometry_msgs/Pose box_pose
geometry_msgs/Pose target_pose
string box_name
